### DISCUSSION TITLE
question about attention maps

### DISCUSSION THREAD

**Discussion by wenjuanyu** [yandex-research/rtdl-revisiting-models#16] [Q&A] [answered]
Thank you so much for the handy tool! I want to check the feature importance from the attention maps and compare it with permutation means. You did this process in section 5.3, could you pls. show the code of this part? or make a simple showcase about how to export the attention map with it~ many thanks in advance~

**Comment by Yura52** [comment, marked as answer]
Hey! Does [this thread](https://github.com/Yura52/tabular-dl-revisiting-models/issues/2) answer your question?

**Reply by wenjuanyu** [reply]
Wieeeii, this is what I looking for~ THX a lot ~
